autopaho/cmd/rpc/.env

```diff
@@ -0,0 +1,10 @@
+export subdemo_serverURL=tcp://127.0.0.1:1883
+export subdemo_clientID=mqtt_subscriber
+export subdemo_topic=topic1
+export subdemo_qos=1
+export subdemo_keepAlive=30
+export subdemo_connectRetryDelay=10000
+export subdemo_writeToStdout="true"
+export subdemo_writeToDisk="false"
+export subdemo_OutputFile="/binds/receivedMessages.txt"
```

Comment:
Did you mean to leave the .env file in the PR? If so I don't think a default like this starting from / is appropriate, even if it's not used by default

Reply:
I did, I thought it may be useful for the RPC example. I've removed it now